cohort-cli/src/test/resources/cql/zip/Breast-Cancer-Screening.xml

```diff
@@ -0,0 +1,298 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Non-blocking: I think these are all copies of the files in cohort-engine?  (I don't see them as moved or deleted there?)

It would be nice to not have duplicate resources in different projects, but I don't know of an established pattern to avoid this.

Reply:
I did some testing trying to figure out how to reuse the test assets from the cohort-engine project via Maven build hooks in the cohort-cli, but the timing was off. I could alternatively have depended directly on filesystem paths cross-project, but that is bad form and introduces a different set of problems. I settled on just duplicating the resources for now.